packages/alias/alias.bench.mjs

```diff
@@ -0,0 +1,54 @@
+import { Suite, chartReport } from 'bench-node';
+import { execPath } from 'node:process';
+import { spawnSync } from 'node:child_process';
+import { fileURLToPath } from 'node:url';
+
+const suite = new Suite({
+	reporter: chartReport,
+});
+
+const cwd = fileURLToPath(import.meta.resolve('./fixtures'));
+const e2eTest = fileURLToPath(import.meta.resolve('./fixtures/e2e.mjs'));
```

Comment:
*very* nit: `cwd` is 95% of `e2eTest`, `import.meta.resolve` and `fileURLToPath` are kind of expensive, and this will only be run in linux (so no need to worry about Windows nonsense)

```suggestion
const e2eTest = fileURLToPath(import.meta.resolve('./fixtures/e2e.mjs'));
const cwd = e2eTest.slice(0, -8);
```